### PR TITLE
Updated text for what-is-github and move to merge with dev

### PR DESCRIPTION
**Pull request by CodeGlitter**
Hi, I made some changes to the text of pt. 1 of the handout. Please check for possible errors and as always, don't hesitate to comment on possible content/formal issues. Thx!

### CODE REVIEW

**Comment by SvetlanaIvanovic** [comment]
Looks good to me. From my view your chapter can be merged as it is with the dev branch.

**Comment by Orock237** [comment]
Good job! Please use the dev-branch

File: README.md [hunk 1]
```diff
@@ -1,2 +1,277 @@
...
+
+**4. Kombination von Git mit IntelliJ/PyCharm: Local Repository und Remote Repository**
+
+**5. Nützliche Git-Tools und Plattformen**
+
+
+
+## Git: Programminformationen und Vorteile bei der Nutzung
+
+[//]: # (Emma)
```
Comment (futurefounder) [inline README.md:16]: This line is not needed and can be deleted

**Comment by futurefounder** [comment]
@CodeGlitter Great overall work, the output is good! I added some formal issues that improve readability of the markdown file for you to consider.

**Comment by futurefounder** [comment]
Thanks for your edits and feedback on the merge conflicts I added the fixes

**Pull request merged**